packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -97,3 +99,11 @@ String debugFormatDouble(double value) {
   }
   return value.toStringAsFixed(1);
 }
+
+/// A setting that can be used to override the platform [Brightness] exposed
+/// from [BindingBase.window]
+///
+/// For an example of using [debugBrightnessOverride], see
```

Comment:
Maybe just say "See also:" since that's our style. You could incorporate this half-sentence into the explanation after the `[WidgetsApp], which` below.